openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -111,10 +109,20 @@ def recurring_nudge_schedule_bin(
             language,
             context,
         )
-        _recurring_nudge_schedule_send.apply_async((site_id, str(msg)), retry=False)
+        with newrelic_function_trace('enqueue_send_task'):
+            _recurring_nudge_schedule_send.apply_async((site_id, str(msg)), retry=False)
+
+
+def _annotate_newrelic_transaction(message_type, site, bin_num, target_day_str):
+    set_custom_metric('message_type', message_type.name)
+    set_custom_metric('site', site.domain)
+    set_custom_metric('bin', bin_num)
+    set_custom_metric('target_day', target_day_str)
```

Comment:
do you think `day_offset` would be more valuable than `target_day` here?  Isn't target_day the datetime string value?

Reply:
Good idea. I'll include both since it would be unwieldy to compute the actual target date from the NR timestamp and day offset.